src/integrationTest/groovy/wooga/gradle/fastlane/IntegrationSpec.groovy

```diff
@@ -26,124 +26,24 @@ import org.junit.contrib.java.lang.system.EnvironmentVariables
-        def subType = (subtypeMatches.matches()) ? subtypeMatches.group("subType") : null
-        type = (subtypeMatches.matches()) ? subtypeMatches.group("mainType") : type
-        switch (type) {
-            case "Closure":
-                if (subType) {
-                    value = "{${wrapValueBasedOnType(rawValue, subType, fallback)}}"
-                } else {
-                    value = "{$rawValueEscaped}"
-                }
-                break
-            case "Callable":
-                value = "new java.util.concurrent.Callable<${rawValue.class.typeName}>() {@Override ${rawValue.class.typeName} call() throws Exception { $rawValueEscaped }}"
-                break
-            case "Object":
-                value = "new Object() {@Override String toString() { ${rawValueEscaped}.toString() }}"
-                break
-            case "Provider":
-                switch (subType) {
-                    case "RegularFile":
-                        value = "project.layout.file(${wrapValueBasedOnType(rawValue, "Provider<File>", fallback)})"
-                        break
-                    case "Directory":
-                        value = """
-                                project.provider({
-                                    def d = project.objects.directoryProperty()
-                                    d.set(${wrapValueBasedOnType(rawValue, "File", fallback)})
-                                    d.get()
-                                })
-                        """.trim().stripIndent()
-                        break
-                    default:
-                        value = "project.provider(${wrapValueBasedOnType(rawValue, "Closure<${subType}>", fallback)})"
-                        break
-                }
-                break
-            case "String":
-                value = "${escapedPath(rawValueEscaped.toString())}"
-                break
-            case "String[]":
-                value = "'${rawValue.collect { it }.join(",")}'.split(',')"
-                break
-            case "File":
-                value = "new File('${escapedPath(rawValue.toString())}')"
-                break
-            case "String...":
-                value = "${rawValue.collect { '"' + it + '"' }.join(", ")}"
-                break
-            case "List":
-                value = "[${rawValue.collect { '"' + it + '"' }.join(", ")}]"
-                break
-            case "Map":
-                value = "[" + rawValue.collect { k, v -> "${wrapValueBasedOnType(k, k.getClass(), fallback)} : ${wrapValueBasedOnType(v, v.getClass(), fallback)}" }.join(", ") + "]"
-                value = value == "[]" ? "[:]" : value
-                break
-            default:
-                value = (fallback) ? fallback.call(type) : rawValue
-        }
-        value
+        // All are cleared now
+        //environmentVariables.clear("FASTLANE_USERNAME", "FASTLANE_PASSWORD")
```

Comment:
what all are cleared now? You can't clear the whole environment if that is what you do now.

Reply:
Not what I do, but what our integration spec does.